src/Fixtures/index.js

```diff
@@ -1271,17 +1261,6 @@ const normalGame = {
   goals: [GoalsFitness.CORE],
   seasonTiming: SeasonTimings.ANYTIME,
   level: Levels.BEGINNER,
-  steps: [
```

Comment:
Isn't it going to break the DrillPage for those drills ?

Reply:
Merci Martin. Cette erreur était surprenante.